Dockerfile

```diff
@@ -8,8 +8,24 @@ RUN mkdir -p /devpi
 
 RUN devpi-init --serverdir /devpi
 
+COPY requirements.txt ./requirements.txt
+COPY start-devpi.sh ./start-devpi.sh
+RUN chmod +x start-devpi.sh
+RUN ./start-devpi.sh
+
+RUN chmod 777 -Rv /devpi/+files
+
+FROM python:3.10-alpine
```

Comment:
I'd keep the same python version, 3.9 was used to start, I would use it again here.